proto/umee/leverage/v1beta1/query.proto

```diff
@@ -27,6 +27,11 @@ service Query {
     option (google.api.http).get = "/umee/leverage/v1beta1/borrowed";
   }
 
+  // BorrowApy queries for the borrow APY of a specified denomination.
+  rpc BorrowApy(QueryBorrowApyRequest) returns (QueryBorrowApyResponse) {
```

Comment:
nit: `BorrowAPY` to this type and the ones below.

Reply:
done at https://github.com/umee-network/umee/pull/348/commits/bf373ca4f5af0cf90793ac5ec92728a436fadf46